### ISSUE TITLE
UI stuck active, flickering and jumps back 10 seconds on alt-tab

### ISSUE THREAD

**Issue by Pentazer** [MediaBrowser/Emby#3664]
When using Emby in Brave Version 1.18.77 Chromium: 87.0.4280.101 (Official Build) (64-bit) the UI flickers rapidly
https://imgur.com/a/6DlTmp0

And when viewing videos the UI is stuck active no matter what I click on the window. It only occasionally goes away if I alt-tab away from emby and then return focus. But if I do, a few seconds later the playback jumps back 10 seconds.
https://imgur.com/a/i81kfse

Here's a copy of the console
`
epad will now require Secure Context. Please update your application accordingly. For more information see https://github.com/w3c/gamepad/pull/120
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
requestAnimationFrame (async)
runInputLoop @ gamepadtokey.js?v=4.5.4.0:1
registrationservices.js?v=4.5.4.0:1 validateFeature: playback
plugin.js?v=4.5.4.0:1 playing url: http://192.168.1.50:8096/emby/videos/60534/master.m3u8?DeviceId=677a9601-60f5-4bdc-a7a9-cbd88b3b8099&MediaSourceId=526e567bd79722259a26a61a0b9261b0&PlaySessionId=a899eb54e41e4ca8b3c4e4a9a4036521&api_key=abac2ca87ff04ac98bd2f3e398b5bfae&VideoCodec=h264&AudioCodec=mp3,aac&VideoBitrate=139889128&AudioBitrate=110872&AudioStreamIndex=1&TranscodingMaxAudioChannels=2&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&ManifestSubtitles=vtt&h264-profile=high,main,baseline,constrainedbaseline,high10&h264-level=52&TranscodeReasons=ContainerNotSupported
DevTools failed to load SourceMap: Could not load content for http://192.168.1.50:8096/web/modules/hlsjs/hls.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
router.js?v=4.5.4.0:1 appRouter - processing path: /web#!/videoosd/videoosd.html, isBack: undefined
router.js?v=4.5.4.0:1 appRouter - user is authenticated
router.js?v=4.5.4.0:1 appRouter - proceeding to /web#!/videoosd/videoosd.html
playbackmanager.js?v=4.5.4.0:1 Active player: {"name":"Web Video Player","id":"htmlvideoplayer","playerName":"Web Video Player","playableMediaTypes":["Video"],"isLocalPlayer":true,"supportedCommands":["GoHome","GoToSettings","VolumeUp","VolumeDown","Mute","Unmute","ToggleMute","SetVolume","SetAudioStreamIndex","SetSubtitleStreamIndex","RefreshMediaSource","SetMaxStreamingBitrate","DisplayContent","GoToSearch","DisplayMessage","SetRepeatMode","PlayMediaSource","PlayTrailers","ToggleFullscreen","PictureInPicture","SetBrightness","SetAspectRatio","SetSubtitleOffset","SetPlaybackRate"]}
playbackorientation.js?v=4.5.4.0:1 error locking orientation: NotSupportedError: screen.orientation.lock() is not available on this device.
router.js?v=4.5.4.0:1 appRouter.sendRouteToViewManager - processing path: isBack: undefined
viewmanager.js?v=4.5.4.0:1 viewManager.onBeforeChange - processing path: isBack: undefined
plugin.js?v=4.5.4.0:1 Setting new text track index to: -1
`

Based on what I see here the issue is my X55 Rhino or the XBONE elite controller. This could be solved if I had a way to disable controller input for Emby.

**Comment by LukePulverenti** [comment]
Hi, did you verify this by disconnecting the controller?

**Comment by Pentazer** [comment]
> Hi, did you verify this by disconnecting the controller?

Yes, disconnecting the controller stops the problem and I can even confirm the rate at which it flickers is based on the axis position of the joystick throttle. Ideally it'd be great if I could disable all controller inputs for Emby as an option.

**Comment by LukePulverenti** [comment]
Hi, are you still seeing this with Emby Server 4.8.11?

**Comment by LukePulverenti** [comment]
Closing, but please feel free to re-open with the requested information. Thanks.
